### PR TITLE
device-scheduling: gpu share with hami

### PR DESCRIPTION
**Pull request by ZiMengSheng**
Support HAMi with GPU Share

### CODE REVIEW

File: docs/user-manuals/device-scheduling-gpu-share-with-hami.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+1. Create a Pod to apply for a GPU card with 50% computing power and 50% video memory, and specify the need for hami-core isolation through the Pod Label koordinator.sh/gpu-isolation-provider
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-example
+  namespace: default
+  labels:
+  	koordinator.sh/gpu-isolation-provider: hami-core
```
Comment (saintube) [inline docs/user-manuals/device-scheduling-gpu-share-with-hami.md:106]: need to fix the indentation
same below

File: docs/user-manuals/device-scheduling-gpu-share-with-hami.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+...
+```
+
+You can find the concrete device allocate result through annotation `scheduling.koordinator.sh/device-allocated`.
+
+2. 通过 kubectl exec 进入 Pod，NVIDIA-SMI 观察 Pod 能够使用的内存上限
```
Comment (saintube) [inline docs/user-manuals/device-scheduling-gpu-share-with-hami.md:153]: in English

File: docs/user-manuals/device-scheduling-gpu-share-with-hami.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+The scheduled GPU devices are bound to the container requires support from the runtime environment. Currently, there are two solutions to achieve this:
+
+| Runtime Environment                           | Installation                                                 |
+| --------------------------------------------- | ------------------------------------------------------------ |
+| Containerd >= 1.7.0 <br /> Koordinator >= 1.6 | Please make sure NRI is enabled in containerd. If not, please refer to [Enable NRI in Containerd](https://github.com/containerd/containerd/blob/main/docs/NRI.md) |
+| others                                        | Please make sure koord-runtime-proxy component is correctly installed in you cluser. If not, please refer to [Installation Runtime Proxy](installation-runtime-proxy). |
+
+#### HAMi-Core Installation
+
+[HAMi-core](https://github.com/Project-HAMi/HAMi-core) is the in-container gpu resource controller, it operates by Hijacking the API-call between CUDA-Runtime(libcudart.so) and CUDA-Driver(libcuda.so).  Actually, the output of HAMI-core build is 'libvgpu.so', we provide a way to distribute it to Nodes.
```
Comment (songtao98) [inline docs/user-manuals/device-scheduling-gpu-share-with-hami.md:31]: "Actually, the output of HAMI-core build is 'libvgpu.so', we provide a way to distribute it to Nodes."

The output from building HAMi-core is `libvgpu.so`. You can directly install HAMi-core on your nodes by deploying the DaemonSet with the YAML file provided below.

**Review by saintube — APPROVED** [review]
/lgtm

**Pull request merged**